daemon/api_systems.go

```diff
@@ -99,6 +103,52 @@ func getSystems(c *Command, r *http.Request, user *auth.UserState) Response {
 	return SyncResponse(&rsp)
 }
 
+type oneSystemResponse struct {
+	// First part is designed to look like `client.System` - the
+	// only difference is how the model is represented
+	Current bool                   `json:"current,omitempty"`
+	Label   string                 `json:"label,omitempty"`
+	Model   map[string]interface{} `json:"model,omitempty"`
+	Actions []client.SystemAction  `json:"actions,omitempty"`
```

Comment:
to be a superset of client.System, this is missing:

```
 // Brand information
Brand snap.StoreAccount
```

which then the spread test should cover